src/components/Separator/Separator.scss

```diff
@@ -1,6 +1,18 @@
 .separator {
     border: 0;
     background: var(--color-gray40);
+
```

Comment:
what if I want to have my separator to be invisible when there no item before OR after ?

Reply:
well I did not think of this one

Reply:
i think it should be the default behaviour :-)

Reply:
Humm nope, because we do have cases in moonstone that expect the separator to show when it's the only child or the first-child, so if we do that we will have to change the implementation of the primary nav :/

Reply:
But I'm adding a fourth option as we speak to handle your cases, not as the default one tho

Reply:
ok, so by default no invisible

Reply:
yes, by default always visible